### PR TITLE
Docs: faster onboarding, identify running processes, improve internal proxy clarification

### PR DESCRIPTION
**Pull request by fredvd**

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: docs/source/configuration/internalproxy.md [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
-While in development, Volto has an internal proxy to the backend API enabled by default.
 It provides a better developer experience out of the box, so the developer doesn't has to
 deal with [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) and can focus on
 develop/test drive/demo Volto.
 
+To understand the need for the internal proxy, there are three processes running in a Volto website:
+
+1. A frontend web application running in your browser (Javascript)
+2. A Node.js server process that delivers the javascript to the client and does
+   Server Side Generation (SSR) of your pages on first request (Javascript, the
```
Comment (sneridagh) [inline docs/source/configuration/internalproxy.md:14]: Watch out, SSG is another thing (what NextJS does). So:

"...does Server Side Rendering (SSR)..."

File: docs/source/configuration/internalproxy.md [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
+
 ## Configuration
 
-By default, Volto expects a Plone backend located at `http://localhost:8080/Plone`.
-These are the settings that allows you to configure the API and how the proxy works:
+The default values from Volto configuration expect a Plone content backend located at `http://localhost:8080/Plone`. Below are the settings that allows you to change the its location
 
-- `apiPath` [API_PATH] - URL of the backend, used through Volto. By default, the proxy URL.
-- `devProxyToApiPath` - The real backend URL, used by the proxy. By default, `http://localhost:8080/Plone`
+- `apiPath` [API_PATH] - URL of the backend, used by Volto running in the frontend browser. By default, the proxy URL. (http://localhost:3000/++api++/)
```
Comment (sneridagh) [inline docs/source/configuration/internalproxy.md:22]: This changed overtime, `apiPath` in the config is no longer meant to be "touched" but controlled via environment variables. 

Also, the apiPath is no longer used in the proxy...

So I would rewrite the entire section using only `devProxyToApiPath` and/or RAZZLE_DEV_PROXY_API_PATH.

File: docs/source/configuration/internalproxy.md [hunk 3]
```diff
@@ -55,6 +71,11 @@ or use the environment variable:
 RAZZLE_DEV_PROXY_API_PATH= RAZZLE_API_PATH=http://localhost:8081/mysite yarn start
 ```
 
+If you disable the proxy, the frontend Volto javascript process in the browser will directly query
```
Comment (sneridagh) [inline docs/source/configuration/internalproxy.md:74]: I think I removed the section "disabling the proxy".

File: docs/source/getting-started/install.md [hunk 4]
```diff
@@ -5,26 +5,45 @@
...
 Depending on the OS that you are using some of the following might change, they
 are assuming a MacOS/Linux machine:
 
+## Components / Processes running
+
+There are three processes continuously running when you have a working Volto website:
+
+1. A frontend web application running in your browser (Javascript)
+2. A Node.js server process that delivers the javascript to the client and does
+   Server Side Generation (SSR) of your pages on first request (Javascript, the
```
Comment (sneridagh) [inline docs/source/getting-started/install.md:22]: Same as above!

File: docs/source/getting-started/install.md [hunk 5]
```diff
@@ -102,7 +119,7 @@ should not throw an error and show the current running containers.
 
 ## Run a Volto ready Plone Docker container
 
-When you have installed Docker, you can run an standard Plone Docker container with the proper configuration for Volto using the `plone.volto` add'on right away by issuing:
+When you have installed Docker, you can use the official Plone Docker container with the proper configuration for Volto using the `plone.volto` add'on right away by issuing:
 
 ```shell
 docker run -it --rm --name=plone \
```
Comment (sneridagh) [inline docs/source/getting-started/install.md:125]: Meh, this one is outdated (see README ones), they use the pip based ones already.

**Review by sneridagh — CHANGES_REQUESTED** [review]
LGTM after fixing issues! Thanks for taking care!

**Comment by sneridagh** [comment]
uh, I now see the merging issues. That's why I saw differences with the master ones.

File: docs/source/configuration/backend.md [hunk 6]
```diff
@@ -1,9 +1,8 @@
 # Backend configuration
 
 ## plone.volto
-In order to fully support all Volto features, the Plone backend API needs to be prepared
-for Volto. The add'on `plone.volto` does all the heavy lifting for you and is ready to use
-in your own projects. We used it in our Getting Started section.
+<<<<<<< HEAD
```
Comment (tiberiuichim) [inline docs/source/configuration/backend.md:4]: Broken git merge artifact.

File: docs/source/configuration/internalproxy.md (hunk 1 above)
Comment (fredvd) [reply docs/source/configuration/internalproxy.md]: fixed

File: docs/source/configuration/internalproxy.md (hunk 3 above)
Comment (fredvd) [reply docs/source/configuration/internalproxy.md]: removed the paragraph on disabling

File: docs/source/configuration/backend.md (hunk 6 above)
Comment (fredvd) [reply docs/source/configuration/backend.md:4]: fixed.  I was still editing that chapter yesterday. ;-)   (see PR label)

File: docs/source/getting-started/install.md (hunk 5 above)
Comment (fredvd) [reply docs/source/getting-started/install.md:126]: I'm not up to date as to which image with which plone version are already using the plone-backend / plone-fontend ones.  :-/

**Comment by fredvd** [comment]
Fixed the merge conflict this weekend, enough small text tinkering for now.  Quick review and then merge? I see another PR/mentionning of the grander docs integration  as well coming up.

**Comment by sneridagh** [comment]
@fredvd watch out, we are about to merge https://github.com/plone/volto/pull/3001 !

**Comment by fredvd** [comment]
@sneridagh I'm not merging my own PR's. But this one is ready.

**Comment by sneridagh** [comment]
@fredvd ok, let's do this, since the merge looks good, let's merge this one. Then https://github.com/plone/volto/pull/3001

/cc @stevepiercy

**Comment by stevepiercy** [comment]
This looks good to me.

#3001 has more work ahead and is not ready for release, unless you don't care about docs that will not automatically build and deploy to production.

There will need to be some merging, cherry-picking, or rebasing, no matter what. I'd suggest having two deployments in parallel, and port doc changes from `master` to `plone6-docs`. Please let me know.

**Pull request merged**